content/en/dashboards/scheduled_reports.md

```diff
@@ -73,7 +73,7 @@ From the configuration modal that opens, you can pause an existing report or cre
 
 ## Permissions
 
-Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles. 
+Users with the Admin role or `Org Management` permission can enable or disable scheduled reports for an account from the **Settings** tab under [Public Sharing][14] in **Organization Settings**. Only users with the **Dashboard Report Write** permission can generate a report. This permission is turned on by default for Admins and off for all other roles.
```

Comment:
could we move this new sentence to the end of this section? the flag being toggled is far rarer than a user missing admin permissions, so it should come at the end of the doc.

and can we add an image to make it super clear what users should be looking for? this is from demo org and safe to use https://a.cl.ly/X6uQdQ1G